src/flux/media.coffee

```diff
@@ -2,6 +2,26 @@ _ = require 'underscore'
 htmlparser = require 'htmlparser2'
 {makeSimpleStore} = require './helpers'
 
+LINKS_BEGIN = ['#']
+LINKS_CONTAIN = ['/contents/', '.cnx.org']
```

Comment:
Huh RE "tries to get if from the tutor-be"? Do you mean the JS tries to get stuff from tutor-be unless it starts with `#` or `https?:` ? Or does JS parse the string `"http://cnx.org/contents/[UUID]"`, pull out `[UUID]`, and ask `/api/books/#/pages/[UUID]`?